cmd/podman/registry/config_tunnel.go

```diff
@@ -2,6 +2,13 @@
 
 package registry
 
+import (
+	"os"
+)
+
 func init() {
 	abiSupport = false
+
+	// Enforce that podman-remote == podman --remote
+	os.Args = append(os.Args, "--remote")
```

Comment:
Is there a potential problem if we duplicate this? Should we only append if --remote is not already set?

Reply:
duping, doesn't hurt. This is faster than searching the list and appending.